ca_gazebo/worlds/empty.world

```diff
@@ -14,7 +14,7 @@
     <physics type="ode">
       <real_time_update_rate>1000.0</real_time_update_rate>
       <max_step_size>0.001</max_step_size>
-      <real_time_factor>1</real_time_factor>
+      <real_time_factor>10</real_time_factor>
```

Comment:
Why did you change this?

Reply:
I wanted the robot to go faster, but just for some testing. Gonna return it to the original state.

Reply:
Consider using `gz physics -u ...` from the terminal